python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -271,7 +271,7 @@ def edge_betweenness_centrality(
         # overwrite the df with the sum of the values for all alike src/dst
         # vertex pairs, resulting in half the edges of the original df from the
         # symmeterized graph.
-        df = df.groupby(by=["src", "dst"]).sum().reset_index()
+        df = df.groupby(by=["src", "dst"], sort=True).sum().reset_index()
```

Comment:
I would be surprised if this requires `sort=True`.  If you specify `sort=False` do any of the BC python tests fail?